### DISCUSSION TITLE
When running `yarn devserver-hot core,coach` gives error `ModuleNotFoundError: No module named 'le_utils.uuidv5'`

### DISCUSSION THREAD

**Discussion by muditchoudhary** [learningequality/kolibri#11252] [Q&A] [answered]
When running `yarn devserver-hot core,coach` I got an error `ModuleNotFoundError: No module named 'le_utils.uuidv5'`

pyenv version : `pyenv 2.3.24`
Python version after activating pyenv : `Python 3.9.9`

Error logs can be found here: https://gist.github.com/muditchoudhary/d2b14f1facb7057bee1fd72d58ed1b62
(First logs after running `yarn run devserver-hot core,coach` second logs after running `pip install -r requirements.txt --upgrade`)

More discussion can be found here: https://github.com/learningequality/kolibri/issues/10993#issuecomment-1717217840

# Solution
1. As mentioned in this https://stackoverflow.com/questions/65640476/pip3-on-python3-9-fails-on-htmlparser-object-has-no-attribute-unescape/65640477#65640477 solution description. Need to upgrade the setup tools.

2. Need to again run (ran first time during setup) the pip commands that are given here: https://kolibri-dev.readthedocs.io/en/develop/getting_started.html#install-python-dependencies
No need to run the **optional** commands.

**Comment by MisRob** [comment, marked as answer]
Answered by @muditchoudhary above (it seems there's no way to mark the initial post of a discussion as answer). Thank you!
